src/sider/historikk/container/HistorikkContainer.tsx

```diff
@@ -144,6 +147,12 @@ function manglendemedvirkningText(
   }
 }
 
+function friskTilArbeidText(friskTilArbeid: VedtakResponseDTO): string {
+  const fom = tilLesbarDatoMedArUtenManedNavn(friskTilArbeid.fom);
+  const tom = tilLesbarDatoMedArUtenManedNavn(friskTilArbeid.tom);
+  return `${friskTilArbeid.veilederident} fattet § 8-5 vedtak. Periode: ${fom} - ${tom}`;
```

Comment:
Litt i tråd med den andre kommentaren min, burde vi skrive 8-5 eller skrive ut hele navnet? 
Feks: `...fattet vedtak for § 8-5 Friskmelding til arbeidsformidling. Periode ...`

Reply:
Eller fjerna det helt? Tagen sier noe om hvilken hendelse det er. 
`(...) fattet vedtak. Periode (...)`